<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weather App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(162, 109, 247);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="main_combo_city">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>- City -</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Berlin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brussels</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ankara</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Istanbul</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="main_combo_country">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>- Country -</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Belgium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Germany</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>United States of America</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="main_btn_search">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Search City</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_lbl_headingcountrytop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Country :</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_lbl_mainheading">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome!</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_lbl_headingcity">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>City:</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_lbl_showcityname">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>City name</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_lbl_showweathericon">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Weather icon</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_lbl_showtemperature">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>341</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="main_lbl_headingpopulation">
       <property name="text">
        <string>Population</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="main_lbl_headingstate">
       <property name="text">
        <string>State / Region</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="main_lbl_showpopulation">
       <property name="text">
        <string>900.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="main_lbl_headingcountry">
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="main_lbl_showcountry">
       <property name="text">
        <string>USA</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="main_lbl_showstate">
       <property name="text">
        <string>Arkansas</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="main_btn_exit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>660</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>321</width>
      <height>511</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>City</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Region/State</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
